### PR TITLE
Add Potato65 Hotswap keyboard

### PR DESCRIPTION
**Pull request by Aeonstrife**
## Description

Added support for Potato65 Hotswap keyboard.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR
- Adds support for the Potato65 hotswap keyboard.

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/buildakb/potato65hs/info.json [hunk 1]
```diff
@@ -0,0 +1,84 @@
+{
+    "keyboard_name": "Potato 65 Hotswap",
+    "url": "https://github.com/Aeonstrife/potato65hotswap",
+    "maintainer": "Maelkk",
+    "width": 16,
+    "height": 5,
+    "layouts": {
+        "LAYOUT_65_ansi": {
```
Comment (fauxpark) [inline keyboards/buildakb/potato65hs/info.json:8]:
```suggestion
        "LAYOUT_65_ansi_blocker": {
```

File: keyboards/buildakb/potato65hs/readme.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Potato65 Hotswap Keyboard
+
+![Potato65 Hotswap](https://i.imgur.com/ysAGZCm.jpg)
```
Comment (fauxpark) [inline keyboards/buildakb/potato65hs/readme.md:3]:
```suggestion
![Potato65 Hotswap](https://i.imgur.com/ysAGZCmh.jpg)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/buildakb/potato65hs/info.json (hunk 1 above)
Comment (Aeonstrife) [reply keyboards/buildakb/potato65hs/info.json]: Rectified the layout. Thank you.

File: keyboards/buildakb/potato65hs/config.h [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+/* key matrix pins */
+#define MATRIX_ROW_PINS { F5, F4, F6, F0, D2 }
+#define MATRIX_COL_PINS { D3, D4, D6, D7, B4, B5, B6, F1, B0, B1, B2, B3, B7, D0, D1 }
+#define UNUSED_PINS
+
+/* COL2ROW or ROW2COL */
+#define DIODE_DIRECTION COL2ROW
+
+/* Set 0 if debouncing isn't needed */
+#define DEBOUNCING_DELAY 5
```
Comment (fauxpark) [inline keyboards/buildakb/potato65hs/config.h:39]:
```suggestion
#define DEBOUNCE 5
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

File: keyboards/buildakb/potato65hs/config.h [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4a56
```
Comment (drashna) [inline keyboards/buildakb/potato65hs/config.h:20]:
```suggestion
#define VENDOR_ID       0x4A56
```

File: keyboards/buildakb/potato65hs/config.h [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+#define RGB_DI_PIN D5
+#ifdef RGB_DI_PIN
+#define RGBLIGHT_ANIMATIONS
+#define RGBLED_NUM 16
+#define RGBLIGHT_HUE_STEP 8
+#define RGBLIGHT_SAT_STEP 8
+#define RGBLIGHT_VAL_STEP 8
+#define RGBLIGHT_LIMIT_VAL 240
+#define RGBLIGHT_SLEEP
```
Comment (drashna) [inline keyboards/buildakb/potato65hs/config.h:59]: Just a nit pick.
```suggestion
#    define RGBLIGHT_ANIMATIONS
#    define RGBLED_NUM 16
#    define RGBLIGHT_HUE_STEP 8
#    define RGBLIGHT_SAT_STEP 8
#    define RGBLIGHT_VAL_STEP 8
#    define RGBLIGHT_LIMIT_VAL 240
#    define RGBLIGHT_SLEEP
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/buildakb/potato65hs/config.h (hunk 5 above)
Comment (Aeonstrife) [reply keyboards/buildakb/potato65hs/config.h]: Resolved. Thank you :)

File: keyboards/buildakb/potato65hs/readme.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+![Potato65 Hotswap](https://i.imgur.com/ysAGZCmh.jpg)
+
+A hot-swappable gasket mount stacked acrylic 65% keyboard
+
+-   Keyboard Maintainer: [Maelkk](https://github.com/Aeonstrife)
+-   Hardware Supported: Potato65 Keyboard
+-   Hardware Availability: Private Group-Buy
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/buildakb/potato65hs/readme.md:11]: Could you add instructions on how to jump to the bootloader?

Comment (Aeonstrife) [reply keyboards/buildakb/potato65hs/readme.md:11]: Added instructions on how to switch to bootloader mode.

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**